docs/lesson-plans/week-1.md

```diff
@@ -0,0 +1,742 @@
+# Week 1: Project Setup and Terminal Basics
```

Comment:
Move homepage from week 5 to week 1 and set up with links to slides, videos (later), and other stuff

Reply:
important links here too